tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -0,0 +1,1209 @@
+===========
```

Comment:
Why did this file grow so much?

Reply:
Rebasing on master removed the bulk of the non-Java-related additions. The remaining Java-related metac/javacp differences are primarily cause by

- known accessibility bug in metac #1325 
- type signature upper/lower bounds conflicts between `java.lang.Object#` and `scala.Nothing#`, I opened https://github.com/scalameta/scalameta/issues/1650 to track this
- metac not being able to preserve parameter names of java methods due to scalac limitation, I opened https://github.com/scalameta/scalameta/issues/1651
- metac uses structural + with types to represent Java intersection types (`A with B` instead of `A & B`). I opened https://github.com/scalameta/scalameta/issues/1652
- wildcard types are unhandled in javacp, I opened https://github.com/scalameta/scalameta/issues/1653